sdk/storage/azfile/fileerror/error_codes.go

```diff
@@ -34,71 +34,72 @@ func HasCode(err error, codes ...Code) bool {
-	InvalidFileOrDirectoryPathName       Code = "InvalidFileOrDirectoryPathName"
-	InvalidHTTPVerb                      Code = "InvalidHttpVerb"
-	InvalidHeaderValue                   Code = "InvalidHeaderValue"
-	InvalidInput                         Code = "InvalidInput"
-	InvalidMD5                           Code = "InvalidMd5"
-	InvalidMetadata                      Code = "InvalidMetadata"
-	InvalidQueryParameterValue           Code = "InvalidQueryParameterValue"
-	InvalidRange                         Code = "InvalidRange"
-	InvalidResourceName                  Code = "InvalidResourceName"
-	InvalidURI                           Code = "InvalidUri"
-	InvalidXMLDocument                   Code = "InvalidXmlDocument"
-	InvalidXMLNodeValue                  Code = "InvalidXmlNodeValue"
-	MD5Mismatch                          Code = "Md5Mismatch"
-	MetadataTooLarge                     Code = "MetadataTooLarge"
-	MissingContentLengthHeader           Code = "MissingContentLengthHeader"
-	MissingRequiredHeader                Code = "MissingRequiredHeader"
-	MissingRequiredQueryParameter        Code = "MissingRequiredQueryParameter"
-	MissingRequiredXMLNode               Code = "MissingRequiredXmlNode"
-	MultipleConditionHeadersNotSupported Code = "MultipleConditionHeadersNotSupported"
-	OperationTimedOut                    Code = "OperationTimedOut"
-	OutOfRangeInput                      Code = "OutOfRangeInput"
-	OutOfRangeQueryParameterValue        Code = "OutOfRangeQueryParameterValue"
-	ParentNotFound                       Code = "ParentNotFound"
-	ReadOnlyAttribute                    Code = "ReadOnlyAttribute"
-	RequestBodyTooLarge                  Code = "RequestBodyTooLarge"
-	RequestURLFailedToParse              Code = "RequestUrlFailedToParse"
-	ResourceAlreadyExists                Code = "ResourceAlreadyExists"
-	ResourceNotFound                     Code = "ResourceNotFound"
-	ResourceTypeMismatch                 Code = "ResourceTypeMismatch"
-	ServerBusy                           Code = "ServerBusy"
-	ShareAlreadyExists                   Code = "ShareAlreadyExists"
-	ShareBeingDeleted                    Code = "ShareBeingDeleted"
-	ShareDisabled                        Code = "ShareDisabled"
-	ShareHasSnapshots                    Code = "ShareHasSnapshots"
-	ShareNotFound                        Code = "ShareNotFound"
-	ShareSnapshotCountExceeded           Code = "ShareSnapshotCountExceeded"
-	ShareSnapshotInProgress              Code = "ShareSnapshotInProgress"
-	ShareSnapshotOperationNotSupported   Code = "ShareSnapshotOperationNotSupported"
-	SharingViolation                     Code = "SharingViolation"
-	UnsupportedHTTPVerb                  Code = "UnsupportedHttpVerb"
-	UnsupportedHeader                    Code = "UnsupportedHeader"
-	UnsupportedQueryParameter            Code = "UnsupportedQueryParameter"
-	UnsupportedXMLNode                   Code = "UnsupportedXmlNode"
+	AccountAlreadyExists                  Code = "AccountAlreadyExists"
```

Comment:
Is this an artifact of regenerating?

Reply:
This diff is because of indentation. I have added an error code, `FileOAuthManagementApiRestrictedToSrp` which is not present in the generated code.